src/apis/clubApi.ts

```
@@ -1,31 +1,14 @@
-import { axiosInstance } from "./axiosInstance";
+import type { ClubDto, CreateClubRequestDto } from "../types/bookClub";
 import type { ClubDto } from "../types/dto";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ClubDto'. Consider to delete it or rename it.

'ClubDto' is defined here:


(lint/suspicious/noRedeclare)